graphql/resolvers/users.js

```diff
@@ -1083,5 +1130,45 @@ module.exports = {
         return user;
       }
     },
+
+    async updateYears() {
+      var users = await User.find();
+      users.forEach(async function(user){
+        //TESTING: Take entire block out of this IF to test on all users
+        if(user.updatedAt){
+          console.log("updatedAt exists....")
+          const email = user.email;
+          const currDate = new Date();
+          var updatedAt = currDate;
+          if(user.updatedAt) updatedAt = new Date(user.updatedAt);
```

Comment:
It'll need to be ran with the check the first time around since no user has it yet and it will throw an error. After running it once, everyone will have it and we can remove the check.